jinahub/indexers/SimpleIndexer/simple_indexer.py

```diff
@@ -93,10 +94,11 @@ def search(
         top_k = int(parameters.get('top_k', self.default_top_k))
         flat_docs.match(
             self._docs,
-            metric=lambda q_emb, d_emb, _: self.distance(
+            metric=lambda q_emb, d_emb: self.distance(
```

Comment:
I believe the metric function in the latest jina version (2.0.17) actually requires 3 arguments, which explains why the unit test is currently failing.

Reply:
Yes I had some kind of a version conflict so - this is no longer required - i will remove it